backtrace-library/src/main/java/backtraceio/library/interfaces/Api.java

```diff
@@ -1,22 +1,38 @@
 package backtraceio.library.interfaces;
```

Comment:
Added question about above about exposing interface method to end-user - wouldn't it be better if a user will use metrics object instead?